src/components/DndTable/EnhancedTableToolbar.js

```diff
@@ -96,6 +100,7 @@ let EnhancedTableToolbar = (props) => {
               <IconButton aria-label="Filter list">
                 <Search onClick={() => setSearch(true)} />
                 {title === "Profile Items" ? <AddModal /> : null}
+                {title === "Users" ? <EditUserModal name="ADD" /> : null}
```

Comment:
Store this types of static values in constants.js file.

Reply:
done